### PR TITLE
Linux event log

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

**Comment by miczyg1** [comment]
@krystian-hebel at the end of the post I would write that these fixes assure the correct operation of TPM TIS interface in the landing-zone and Linux kernel as per requirement: "<requirement>"

or something like that

**Comment by krystian-hebel** [comment]
> @krystian-hebel at the end of the post I would write that these fixes assure the correct operation of TPM TIS interface in the landing-zone and Linux kernel as per requirement: ""
@miczyg1 are those requirements publicly mentioned anywhere? Readers may not know what exactly is on that list.

**Comment by miczyg1** [comment]
@krystian-hebel maybe they are not, but note that these blog posts are not only for common readers but for the people who will verify our work too.

File: blog/content/post/2020-08-10-trenchboot-event-log.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+with `cbmem` tool.
+
+The following instructions assume TPM2.0, for TPM1.2 change `sha256` to `sha1`
+in the commands. The output format of the event log is also slightly different.
+If in doubt, refer to the previous post.
+
+This is the example log:
+
+```
+$ ./cbmem -d
```
[the remark this answers is not in the document]
Comment (krystian-hebel) [reply blog/content/post/2020-08-10-trenchboot-event-log.md:45]: Done

File: blog/content/post/2020-08-10-trenchboot-event-log.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+  9 : 0x0000000000000000000000000000000000000000
+  10: 0x0000000000000000000000000000000000000000
+  11: 0x0000000000000000000000000000000000000000
+  12: 0x0000000000000000000000000000000000000000
+  13: 0x0000000000000000000000000000000000000000
+  14: 0x0000000000000000000000000000000000000000
+  15: 0x0000000000000000000000000000000000000000
+  16: 0x0000000000000000000000000000000000000000
+  17: 0x75AAEE1F5F811BA64B45B45C68F34281E9A99AE6
+  18: 0x0000000000000000000000000000000000000000
```
[the remark this answers is not in the document]
Comment (krystian-hebel) [reply blog/content/post/2020-08-10-trenchboot-event-log.md]: Added new commit to the kernel, works now

**Pull request merged**